setup.py

```diff
@@ -1,19 +1,148 @@
 import os
 import sys
+import platform
+
+from distutils.core import setup, Extension 
 
 if os.name == 'nt':
     from distutils import msvc9compiler
     msvc9compiler.VERSION = 14.0 #Visual studio 2015
 
-if sys.version > '3':
-    setup_file = "setup_3.py"
```

Comment:
Since python2 is no longer supported, if it runs as python2, it have to exit here, for example:

```
if sys.version <= '2':
    print('No more supports python2, use python3 instead')
    sys.exit(1)
```

Reply:
okay. i'll add the code. thank you.